src/app.js

```diff
@@ -121,6 +121,7 @@ setGTM({
   uaContainerId: APP.GTM.UA_CONTAINER_ID,
   ga4Disabled: APP.GTM.GA4_DISABLED,
   ga4ContainerId: APP.GTM.GA4_CONTAINER_ID,
+  isDataSensitive: APP.GTM.ANALYTICS_DATA_SENSITIVE,
```

Comment:
SUPER confusing I know, but common express requires this to be called `analyticsDataSensitive` rather than `isDataSensitive`

https://github.com/govuk-one-login/ipv-cri-common-express/pull/376/files#diff-9dd464578f3b1db4bbfb4f1d2e68c13907ac3b617fd72f3390385f8e2bf86289

Reply:
Should have picked that up when I reviewed their PR - sorry!

Reply:
Updated the name now